src/modules/status-bar/status-bar.ts

```diff
@@ -112,4 +132,33 @@ export class StatusBar {
     this.isEncryptedCommunication = protocol === 'https://';
     this.processEngineRoute = route;
   }
+
+  private _disposeAllSubscriptions(): void {
+    this._subscriptions.forEach((subscription: Subscription) => {
+      subscription.dispose();
+    });
+  }
+
+  private async _updateStatusBar(): Promise<void> {
+    const solutionUri: string = this._router.currentInstruction.queryParams.solutionUri;
+    const noSolutionUriSpecified: boolean = solutionUri === undefined;
+
+    if (noSolutionUriSpecified) {
+      const remoteSolutionUri: string = window.localStorage.getItem('processEngineRoute');
+      this.activeSolutionEntry = this._solutionService.getSolutionEntryForUri(remoteSolutionUri);
+    } else {
+      this.activeSolutionEntry = this._solutionService.getSolutionEntryForUri(solutionUri);
+    }
```

Comment:
```suggestion
     const uri: string = (noSolutionUriSpecified) 
       ? window.localStorage.getItem('processEngineRoute') 
       : solutionUri;

     this.activeSolutionEntry = this._solutionService.getSolutionEntryForUri(uri);
```

With this, we get rid of the if-clause.